packages/uicore/src/components/MultiStepProgressIndicator/MultiStepProgressIndicator.tsx

```diff
@@ -9,23 +9,43 @@ import React from 'react'
 import cx from 'classnames'
 
 import { Layout } from '../../layouts/Layout'
+import { Text } from '../..'
 
 import css from './MultiStepProgressIndicator.css'
+import { Color } from '@harness/design-system'
 
 type StepStatus = 'TODO' | 'INPROGRESS' | 'FAILED' | 'SUCCESS'
 
 export interface MultiStepProgressIndicatorProps {
-  progressMap: Map<number, StepStatus>
+  progressMap: Map<number, { StepStatus: StepStatus; StepName?: string }>
```

Comment:
usually variable names such as `StepStatus`, except enums should start with a lower-case character.